frontend/ui/widget/imagewidget.lua

```diff
@@ -222,12 +216,11 @@ function ImageWidget:_loadfile()
                 self._bb_disposable = false -- don't touch or free a cached _bb
                 -- cache this image
                 logger.dbg("cache", hash)
-                cache = ImageCacheItem:new{
+                cache = {
                     bb = self._bb,
                     is_straight_alpha = self._is_straight_alpha,
                 }
-                cache.size = tonumber(cache.bb.stride) * cache.bb.h
-                ImageCache:insert(hash, cache)
```

Comment:
(Not your code, but this item being named `cache` reads ugly/confusing, would have been better named `cached` or `item`.)